src/gc/config.d

```diff
@@ -186,6 +186,47 @@ body
     return true;
 }
 
+bool parse(T:const(char)[])(const(char)[] optname, ref const(char)[] str, ref T res)
+in { assert(str.length); }
+body
+{
+
+    int start;
+    int nscanned;
+
+    for(;start<str.length; start++)
+    {
+        if(!isspace(str[start]))
+            break;
+    }
```

Comment:
The options are already space separated, so this will never do anything useful. There were also 2 helpers `skip!isspace` and `find!isspace` to avoid the loop.